profiling/tests/phpt/allocation_generator_01.phpt

```diff
@@ -13,17 +13,19 @@ restores the opline in a opcode handler!
 --SKIPIF--
 <?php
 if (!extension_loaded('datadog-profiling'))
-    echo "skip: test requires datadog-profiling", PHP_EOL;
+    die("skip: test requires datadog-profiling");
+if (PHP_VERSION_ID <= 80010)
+    die("skip: PHP is buggy");
```

Comment:
What's the deal here for this test?

Reply:
Ah. I mean I guess this is a really rare condition because they have to OOM at a highly specific spot.